svgr.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
       'prefixIds',
     ],
   },
-  svgProps: { role: 'img' },
+  svgProps: { 'aria-hidden': 'true' },
```

Comment:
Since we require a `label` in the `IconButton` (which is the only place Icons are really displayed by themselves) I think this makes sense